Dockerfile

```diff
@@ -7,11 +7,17 @@ RUN apk --update add go git \
   && export PATH=$PATH:$GOPATH/bin \
   && mkdir -p $GOPATH/src/${ORG_PATH} \
   && ln -s ${PWD} $GOPATH/src/${REPO_PATH} \
-  && cd $GOPATH/src/${REPO_PATH} \
   && go get github.com/mailgun/vulcand \
+  # checkout older version of vulcan \
+  && cd $GOPATH/src/github.com/mailgun/vulcand \
+  && git checkout 791285c97cbf28f8eac1ef7222e103990c2d0b08 \
```

Comment:
I can't find any. 
As a separate task, it would be a good idea to try out using the latest vulcan (pre go1.5 & vendoring support) which can be still bundled with the auth middleware.